docs/dev/release.rst

```diff
@@ -342,6 +368,28 @@ Milestones bookkeeping
    will need to move their change log entries to the new release section that you
    have created in ``CHANGES.rst`` during the release process.
 
+.. _release-labels:
+
+Labels bookkeeping
+==================
+
+This is only applicable if you are doing a new branched release.
+In the instructions below, ``A.B`` is the old release and ``A.C`` is
+the new release:
+
+#. Go to `Labels <https://github.com/spacetelescope/jdaviz/labels>`_.
+
+#. Find the old ``backport-vA.B.x`` label. Click its "Edit" button and
+   add ``:zzz: `` in front of it. This would send it all the way to the
```

Comment:
I dunno what is wrong. Maybe it doesn't like the whitespace?
```suggestion
   add ``:zzz:`` in front of it. This would send it all the way to the
```